src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -0,0 +1,77 @@
+{% extends "layout.njk" %}
```

Comment:
That file, I mean `src/main/features/claim/views/eligibility/not-eligible.njk` should not be here, it has been moved to other location.

Reply:
@RonanFitzgerald is there any change in that file or just something added accidentally during merge?

Reply:
@geek-soft 
Yep there was a change - I changed the ineligibility page for UK addresses to remove the paper route and make it clear you can't make a money claim if you don't have a UK address.

Prod should match this page in the prototype: https://hmcts-cmc-prototype.herokuapp.com/dashboard/v4/idam/gov/v2/eligibility/result?a=c-address&radio_group2=claimant_non_uk